source/question_options.md

```diff
@@ -1,37 +1,43 @@
 # Using the Question Options
 
-The Question Options screen contains some very useful features to get the most out of your form. You can click Setting to access the Question Options screen.
+The Question Options screen contains some very useful features to get the most out of your form. You can click the <i class="k-icon k-icon-settings"></i> Settings button to access the Question Options screen.
 
-![image](/images/question_options/options.png)
+![image](/images/question_options/options2.png)
 
 ## Data Column Name
 
 The **Data Column Name** is the unique identifier (ID) of your question.
 
-This field is mandatory for every question. Only letters, numbers, and underscores are allowed in this field, and the field must start with a letter or an underscore. You can input anything you like, such as `what_is_your_name` or `age`.
+This field is mandatory for every question. Only letters, numbers, and underscores are allowed in this field, and the field must start with a letter or an underscore. You can input anything you like, such as `what_is_your_name` or `age`. 
 
 The Data Column Name (ID) is important because it is used in the column headers of tables and spreadsheets after your data has been collected. If you want your spreadsheet to follow a specific naming convention, you should specify the name for each of your questions before deploying the form as a data collection project.
 
 ## Guidance Hint (optional)
 
-**Guidance Hint** are help texts that will be displayed underneath your questions on the form.
+**Guidance Hints** are extra instructions that you can add to your questions as notes. These notes are not normally shown on the form by default.
 
-Hints are optional. The hint text is often used to provide additional instructions to your interviewer staff. For example, your question label might be `How old are you?` and your hint might be `If respondent doesn't know, enter 999`.
+Guidance hints can be used as internal notes when collaborating with others in developing the form. You can also show them on printouts or during training for enumerators.
 
 ## Mandatory Response
 
-If a question is **mandatory**, the interviewer needs to provide an answer in order to finalize the form.
+This settings allows you to specify whether the question must be responded to at all times or not.
```

Comment:
Maybe just note that this is equivalent to the `required` field in XLSForm and link to the docs: https://xlsform.org/en/#required